service-manual/measurement/completion-rate.md

```diff
@@ -20,117 +20,85 @@ breadcrumbs:
     url: /service-manual/measurement
 ---
 
-When users are unable to complete a digital transaction, they can be pushed to use other channels. This leads to low levels of digital take-up and customer satisfaction, and a higher cost per transaction.
+Completion rate shows how many people manage to use the service successfully.
 
-Measuring end-to-end completion rates helps to identify whether users have problems completing a transaction. Analysing the drop-out rate at each step of a transaction will pinpoint the specific processes that users fail to complete.
-
-## What you will be measuring
-
-The completion rate measures the proportion of people who start a transaction and are able to complete it.
-
-A transaction is a self-contained process that's defined by the service manager, and is typically only completed when a user has been through the whole thing end to end.
-
-A transaction completed only partially online means the user may complete a single process within the overall transaction, like booking an appointment or completing a part of an application.
+Once your service goes live you’ll find steps that cause users to give up, and bugs that make the service impossible to complete. And as you fix these issues, your completion rate will rise.
 
 ## How to measure completion rate
 
-1.    Work out the number of completed transactions.
-2.    Divide it by the number of started transactions, expressed as a percentage.
-
-### Before the transaction starts
+1. Count the number of completed transactions
+2. Divide it by the number of started transactions, as a percentage
 
-Before the transaction has started, users should clearly be told:
+## What is a completed transaction?
 
-* what the outcome of the transaction will be
-* who can use it
-* how long it's likely to take
-* what they’ll need to complete it (eg a reference number or credit card)
+A transaction is completed when **the service did what it was supposed to do**.
 
-Also provide users with the eligibility criteria and the costs to complete the transaction. This will help you to reduce dropouts later on in the transaction.
+Often this will be because the user was successful, for example:
 
-Typically the eligibility criteria will be given on a single page. In some cases, there may be a set of pages that checks a user’s eligibility based on information they provide.
+* A user claimed Carer’s Allowance
+* A user paid their vehicle tax
 
-Don’t make it possible for users to bypass your service’s start page via links or search engine results. Users who try to directly access another page in the transaction should be sent back to your start page, unless they are resuming a previously saved transaction.
+Sometimes, even though **the service did what it was supposed to do**, the user won’t be successful. These still count as completed transactions. For example:
```

Comment:
Similar comment about successful. Maybe define earlier on that success means achieving their desired outcome, then reuse the definition?